etc/scripts/includes/pipeline-env.sh

```diff
@@ -67,10 +67,12 @@ if [ -z "${__PIPELINE_ENV_INCLUDED__}" ]; then
 
     if [ -n "${JENKINS_HOME}" ] ; then
         export PIPELINE="true"
-        export JAVA_HOME="/tools/jdk-11.0.12"
+        export JAVA_HOME="/tools/jdk-17"
```

Comment:
Reminder: we will want to upgrade to `jdk-17.0.3` when it is released. It should have the CDS archive fix for issue #3608 (which apparently our jlink package test is not hitting).